cplane-mock/src/main.rs

```diff
@@ -82,16 +83,46 @@ struct WakeComputeQuery {
 #[derive(Serialize)]
 struct WakeComputeResponse {
     pub address: String,
+    pub server_name: Option<String>,
     pub aux: MetricsAuxInfo,
 }
 
 #[derive(Serialize)]
 pub struct MetricsAuxInfo {
     pub endpoint_id: String,
     pub project_id: String,
-    pub branch_id: &'static str,
+    pub branch_id: String,
+    // note: we don't use interned strings for compute IDs.
+    // they churn too quickly and we have no way to clean up interned strings.
+    pub compute_id: String,
+    #[serde(default)]
+    pub cold_start_info: ColdStartInfo,
 }
 
+#[derive(Debug, Default, Serialize, Deserialize, Clone, Copy, FixedCardinalityLabel)]
```

Comment:
Nit: we don't need the FixedCardinalityLabel here, or measured at all (unless you want to include prometheus exports for the mocked cplane)